generators/server/templates/src/main/java/package/Application.java.ejs

```diff
@@ -102,6 +104,8 @@ public class <%= mainClass %> {
      * @param args the command line arguments.
      */
     public static void main(String[] args) {
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
```

Comment:
Maybe we should log the default timezone before changing it?

        ZoneId defaultZoneId = ZoneId.systemDefault();
        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
        log.info("Timezone changed from {} to UTC", defaultZoneId);

Reply:
Added